lib/fireblocks/create_transaction.go

```diff
@@ -0,0 +1,80 @@
+package fireblocks
+
+import (
+	"context"
+	"crypto/sha256"
+	"encoding/hex"
+	"encoding/json"
+	"net/http"
+	"strconv"
+
+	"github.com/omni-network/omni/lib/errors"
+)
+
+type TransactionRequestOptions struct {
+	Message UnsignedRawMessage
+}
+
+func (c Client) CreateTransaction(ctx context.Context, request CreateTransactionRequest) (*TransactionResponse, error) {
+	res := TransactionResponse{}
+
+	jwtToken, err := c.GenJWTToken(transactionEndpoint, request)
+	if err != nil {
+		return nil, err
+	}
+
+	response, err := c.http.SendRequest(
+		ctx,
+		transactionEndpoint,
+		http.MethodPost,
+		request,
+		c.getHeaders(jwtToken),
+	)
+
+	if err != nil {
+		return nil, err
+	}
+
+	err = json.Unmarshal([]byte(response), &res)
+	if err != nil {
+		return nil, errors.Wrap(err, "unmarshaling response")
+	}
+
+	return &res, nil
+}
+
+func NewTransactionRequest(opt TransactionRequestOptions) (CreateTransactionRequest, error) {
+	var rawMessageData RawMessageData
+	h := sha256.New()
+	wrappedMessage := "\x19Ethereum Signed Message:\n" + strconv.Itoa(len(opt.Message.Content)) + opt.Message.Content
```

Comment:
this `\x19Ethereum Signed Message` isn't required I think

Reply:
Their example has it, so I put it in

Reply:
Sure, but do you want transactions that work or that don't? Pretty sure it will not work.